auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -83,7 +105,88 @@
+                  color="primary"
+                  min-width="5rem"
+                  min-height="2rem"
+                  class="open-action-btn"
+                  v-on="on"
+                  @click="action(item)"
+                >
+                  {{ getPrimaryAction(item) }}
+                <v-icon class='external-icon pl-1' v-if="isOpenExternal(item)" small>mdi-open-in-new</v-icon>
+              </v-btn>
+              </template>
+              <span>Go to {{ getTooltipTargetDescription(item) }} to access this business</span>
+            </v-tooltip>
+            <!-- More Actions Menu -->
+            <span class="more-actions">
+              <v-menu
+                :attach="`#action-menu-${index}`"
+                v-model="dropdown[index]"
+              >
+                <template v-slot:activator="{ on }">
+                  <v-btn
+                    small
+                    color="primary"
+                    min-height="2rem"
+                    class="more-actions-btn"
+                    v-on="on"
+                  >
+                    <v-icon>{{dropdown[index] ? 'mdi-menu-up' : 'mdi-menu-down'}}</v-icon>
+                  </v-btn>
+                </template>
+                <v-list>
+                  <v-list-item
+                    v-if="showOpenButton(item)"
+                    class="actions-dropdown_item my-1"
+                    data-test="use-name-request-button"
+                    @click="goToNameRequest(item.nameRequest)"
+                  >
+                    <v-list-item-subtitle>
+                      <v-icon small>mdi-format-list-bulleted-square</v-icon>
+                      <span class="pl-1">Open Name Request</span>
+                    </v-list-item-subtitle>
+                  </v-list-item>
+                  <v-list-item
+                    v-if="showRemoveButton(item)"
+                    class="actions-dropdown_item my-1"
+                    data-test="remove-button"
+                    @click="removeBusiness(item)"
+                  >
+                    <v-list-item-subtitle v-if="isTemporaryBusiness(item)">
+                      <v-icon small>mdi-delete-forever</v-icon>
+                      <span class="pl-1">Delete {{tempDescription(item)}}</span>
+                    </v-list-item-subtitle>
+                    <v-list-item-subtitle v-else>
+                      <v-icon small>mdi-delete</v-icon>
+                      <span class="pl-1">Remove From Table</span>
+                    </v-list-item-subtitle>
+                  </v-list-item>
+                  <v-list-item
+                    v-if="showAmalgamateShortForm(item)"
+                    class="actions-dropdown_item my-1"
+                    data-test="use-name-request-button"
+                  >
+                    <v-list-item-subtitle>
+                      <v-icon small>mdi-checkbox-multiple-blank-outline</v-icon>
+                      <span class="pl-1">Amalgamate Now (Short Form)</span>
```

Comment:
Please comment in the ticket the scenarios that you now support (including this one) -- for QA.